store/iavlstore_test.go

```diff
@@ -28,8 +28,8 @@ var (
 	nMoreData = 0
 )
 
-// make a tree and save it
-func newTree(t *testing.T, db dbm.DB) (*iavl.MutableTree, CommitID) {
+// make a tree with data from above and save it
+func newAlohaTree(t *testing.T, db dbm.DB) (*iavl.MutableTree, CommitID) {
```

Comment:
Que?

Reply:
indeed, :palm_tree: ?

Reply:
It's full of some random data include the word "aloha".  Differentiates it from a "newTree".